be/src/common/config.h

```diff
@@ -595,6 +595,9 @@ CONF_Bool(ignore_rowset_stale_unconsistent_delete, "false");
 // The chunk size for vector query engine
 CONF_Int32(vector_chunk_size, "4096");
 
+// Mem limit for single chunk
+CONF_Int64(chunk_mem_size, "33554432");
+
```

Comment:
may be small for large memory hosts.

Reply:
It should not to be large, the memory usage is channel * dop * chunk_mem_size